.github/workflows/playwright.yml

```
@@ -14,12 +13,33 @@ jobs:
           distribution: temurin
-          cache: maven
+
+      - name: Cache Maven local repository
+        id: cache-maven
+        uses: actions/cache@v4
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-m2-
+          save-always: true
```

actionlint (1.7.11)

[error] avoid using deprecated input "save-always" in action "actions/cache@v4": save-always does not work as intended and will be removed in a future release. A separate `actions/cache/restore` step should be used instead. See https://github.com/actions/cache/tree/main/save#always-save-cache for more details

(action)